stubs/dateparser/dateparser/languages/locale.pyi

```diff
@@ -3,6 +3,7 @@ from re import Pattern
 from dateparser.conf import Settings
 
 NUMERAL_PATTERN: Pattern[str]
+ALWAYS_KEEP_TOKENS: list[str]
```

Comment:
This is just a re-export of https://github.com/python/typeshed/blob/c6126b5f52c9e1ad294f7435d1a2242c2f14103a/stubs/dateparser/dateparser/languages/dictionary.pyi#L5

I think we can allowlist it

```suggestion
```